packages/core/ui/NewSessionCards.js

```diff
@@ -93,10 +99,48 @@ ProceedEmptySession.propTypes = {
 
 export function AddLinearGenomeViewToSession({ root }) {
   const launchLGV = () => {
-    // console.log('add LGV', root)
-    root.setDefaultSession()
-    // console.log('add LGV session', root.session)
+    // console.log('add lgv', root)
+    // root.session.addView('LinearGenomeView', {})
+    // const snapshot = getSnapshot(root?.session)
+    // console.log(snapshot)
+    // root.addSavedSession({ name: snapshot.name })
+
+    // // localStorage.setItem(snapshot.id, { session: root.session })
+    // localStorage.setItem(
+    //   `autosave-${root.configPath}`,
+    //   JSON.stringify({
+    //     session: {
+    //       ...snapshot,
+    //       name: `${snapshot.name}-autosaved`,
+    //     },
+    //   }),
+    // )
+    // sessionStorage.setItem('current', JSON.stringify({ session: root.session }))
+    // root.setSession(snapshot)
+    // console.log(root)
+    console.log('add lgv', root)
     root.session.addView('LinearGenomeView', {})
+    const snapshot = getSnapshot(root?.session)
+    console.log(snapshot)
+
+    // root.addSavedSession({ name: snapshot.name })
+
+    localStorage.setItem(
+      `autosave-${root.configPath}`,
+      JSON.stringify({
+        session: {
+          ...snapshot,
+          name: `${snapshot.name}-autosaved`,
+        },
```

Comment:
there are autoruns on the rootmodel that will do this localstorage setItem automatically for both autosave- and current

Reply:
Ahhh noted, will fix that.